frontend/src/App.js

```diff
@@ -16,9 +17,17 @@ function App() {
   return (
     <div className="app">
       <Header searchHandler={handleSearch} />
-      {searchResults && <p>{searchResults.results.map(result => <p>{result.name}</p>)}</p>}
+      {!searchResults ? (
+        <InfoPage />
+      ) : (
+        <div className="search-results">
+          {searchResults.results.map(result => (
+            <p key={result.id}>{result.name}</p>
+          ))}
+        </div>
+      )}
```

Comment:
I'm really excited that you came up with a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator) here but don't display the info page on `App.jsx` for now. 

We'll probably need to set up different pages so in order to avoid dealing with a bunch or merge conflicts on the way we'll do the connecting when we have all the parts ready for a feature. 

This was a nice implementation though - it's just not likely to be the last and it's already got some conflicts with the updated dev since you made the pull request, so let's keep it simple for ourselves and not edit the `App.jsx` file on this one